### DISCUSSION TITLE
Indexing Failed: DNS server unresponsive

### DISCUSSION THREAD

**Discussion by HolmesSoftware** [vercel/community#2611] [Help] [answered]
### Summary

I'm using the Vercel Nameservers, and everything was working correctly. 

Starting two days ago, I have been unable to index any of my new pages. All the pages provide one of the following responses in Google Search Console:

- Failed: Robots.txt unreachable
- Failed: DNS server unresponsive

This cant be because of a code change since I tested it on other projects where the code hasn't been updated in over two months, and they are now also failing.

I have ran DNS tests on and everything comes back looking good: [intodns report](https://intodns.com/animetrivia.app)

These are the nameservers I'm using:
- ns1.vercel-dns.com
- ns2.vercel-dns.com

![image](https://github.com/vercel/community/assets/54856549/693611a6-35a0-4308-9aff-8a7b6005f478)

### Steps to Reproduce

Open Google Search Console.

Go to any URL using the "**Inspect any URL**" search bar

Hit: **Test Live URL** button on the right side.

**Comment by amyegan** [comment]
Thanks for reporting this problem, @HolmesSoftware. I shared it with our network team for investigation, and we'll post an update when we have more info

**Reply by amyegan** [reply]
This isn't solved yet, but we opened a new incident report that you can follow for the latest updates: https://www.vercel-status.com/incidents/8w42qmnv83cj

**Reply by HolmesSoftware** [reply]
Thank you!

**Comment by andinianst93** [comment]
> ### Summary
> 
> I'm using the Vercel Nameservers, and everything was working correctly. 
> 
> Starting two days ago, I have been unable to index any of my new pages. All the pages provide one of the following responses in Google Search Console:
> 
> - Failed: Robots.txt unreachable
> - Failed: DNS server unresponsive
> 
> This cant be because of a code change since I tested it on other projects where the code hasn't been updated in over two months, and they are now also failing.
> 
> I have ran DNS tests on and everything comes back looking good: [intodns report](https://intodns.com/animetrivia.app)
> 
> These are the nameservers I'm using:
> - ns1.vercel-dns.com
> - ns2.vercel-dns.com
> 
> ![image](https://github.com/vercel/community/assets/54856549/693611a6-35a0-4308-9aff-8a7b6005f478)
> 
> 
> ### Example
> 
> _No response_
> 
> ### Steps to Reproduce
> 
> Open Google Search Console.
> 
> Go to any URL using the "**Inspect any URL**" search bar
> 
> Hit: **Test Live URL** button on the right side.

I also experience this problem. It has been more than a week. I can not request or test live URL in Google Search Console. Please help.

**Comment by BrunoQuaresma** [comment]
Having same issue 🤔

**Comment by BrunoQuaresma** [comment]
@amyegan I see the incident was closed as resolved but I still seeing it:

<img width="924" alt="Screen Shot 2023-05-29 at 08 00 44" src="https://github.com/vercel/community/assets/3165839/9420eeda-d275-42eb-8259-9d11ae7d8a83">

**Comment by BrunoQuaresma** [comment]
Maybe related:
https://support.google.com/webmasters/thread/216797133/availability-fails-due-to-robots-txt-unreachable?hl=en

**Comment by damonphilipross** [comment]
I'm also experiencing this problem intermittently for more than a week now

**Reply by offak** [reply]
Having same issue, thanks for the reporting!

**Comment by swarnava** [comment]
Hi,

Can you get any further information from within Google on how exactly these requests are failing and if they are DNS failures or to do with something else?

If Google are able to provide you with query logs that show the queries being sent to us and failing, we can then check further. We would need to know the locations / IPs that the requests were being sent to us from and failing, so that we can check local pops more closely.

**Reply by damonphilipross** [reply]
![image](https://github.com/vercel/community/assets/26593094/85a8f633-c479-4351-b155-aba3a2841c3d)
I get this error despite my robots.txt being definitely available on my site https://obituaries.nz/robots.txt I also get the same issue DNS server unresponsive. Not sure how to provide you with query logs as this is just using the google search console

**Reply by BrunoQuaresma** [reply]
I understand that logs would be helpful but it is VERY unlikely that Google will provide any sort of logs for regular users 😅

**Comment by offak** [comment]
Unfortunately, there is no chance that we will get any logs from Google. But you can test the DNS here if you do not want to use Search Console
https://search.google.com/test/mobile-friendly

The issue is still persisting

**Comment by gurkerl83** [comment]
Same here!
Update: After removing and re-adding the respective domain from / to the project in the vercel dashboard, it starts working again.

**Reply by offak** [reply]
Thanks for the tip! Fixed the issue for me also.

**Comment by smaeda-ks** [comment, marked as answer]
@gurkerl83 @offak @BrunoQuaresma @damonphilipross @HolmesSoftware @andinianst93 

To everyone in this thread, could you check your latest status and share that with us?

**Reply by smaeda-ks** [reply]
For context: we've been suspecting Google's side as well.
https://support.google.com/webmasters/thread/216797133?hl=en&msgid=218480062

Given it is a DNS issue, it must be happening when GoogleBot is trying to reach our name servers that our upstream provider manages to get the IP address of the given domain name. And this network path is GoogleBot -> the Internet -> our name servers (managed by the upstream provider), so it's not something we can directly influence, unlike our other infrastructure. Regardless, we will deliver updates as we hear more from them.

**Reply by smaeda-ks** [reply]
That said, @gurkerl83 's comment above:

> Update: After removing and re-adding the respective domain from / to the project in the vercel dashboard, it starts working again.

This cannot be a workaround, and it was just a coincidence. It is totally possible that the issue might have been resolved by either of them silently, but hard to be sure at this moment. Therefore, we're asking you to recheck the current status. Thanks for your help

**Reply by damonphilipross** [reply]
working for me now cheers

**Reply by offak** [reply]
Same here - working OK, no errors so far.

**Reply by BrunoQuaresma** [reply]
Working OK! Thank you.

**Reply by smaeda-ks** [reply]
Thanks all for reporting back to us so far. Looks like it might be getting resolved. We still don't know what exactly happened, but will let you know more when we have an update to share.

**Reply by gurkerl83** [reply]
It is still working. I remember from yesterday that the Google Search Console showed outdated data in the live URL check. I have other domains directed to a single project. I did not observe the indexing problem on these domains, only on a brand new one I transferred a couple of days ago from Namecheap to Vercel; maybe this is relevant?

**Reply by smaeda-ks** [reply]
Update: Google has just acknowledged that they found a potential cause that could have been throttling traffic to our name servers on their side. We will work with them to prevent this from happening again in the future. But for now, we consider this issue as resolved.

**Reply by smaeda-ks** [reply]
We've started hearing similar reports again. It's possible that the issue is happening again on the Google end. We're reaching out to Google and will update here.

**Reply by raja-vignesh** [reply]
Yes. it's happening again.
<img width="911" alt="Screenshot 2023-06-09 at 10 07 06" src="https://github.com/vercel/community/assets/3530541/036b628e-6a5f-4ef8-a079-3e6d1373ab47">

**Reply by smaeda-ks** [reply]
This has been mitigated. We're monitoring

**Reply by raja-vignesh** [reply]
still getting the same error

**Reply by smaeda-ks** [reply]
@raja-vignesh what's the crawl time?

**Reply by raja-vignesh** [reply]
@smaeda-ks it's working now. Previously checked at 11:33 AM BST. Thanks!

**Reply by raja-vignesh** [reply]
@smaeda-ks Any idea, what was causing this?

**Comment by tarizer** [comment]
I am having exactly the same issue in the google console
`Failed: Robots.txt unreachable`

**Reply by luisjuarros** [reply]
me too

**Reply by vincentri** [reply]
Any update?

**Reply by smaeda-ks** [reply]
I will take a look

**Reply by ericshively** [reply]
Still having this issue

**Reply by jamesdwalkeruk** [reply]
Same, been intermittently happening since at least mid December. I'm not using Vercel's nameservers though, it's Google Domains with the Vercel CNAME and A record. And not getting the `Failed: DNS server unresponsive` error, only `Failed: Robots.txt unreachable`.

**Reply by valentinjadot** [reply]
Sema here! Failed: Robots.txt
